README.md

```diff
@@ -1,10 +1,14 @@
 # Dock Wallet SDK
 
-Using [polkadot-js](https://polkadot.js.org/) libraries in React Native is a challenge, due to a lack of WebAssembly support.
+The [Wallet SDK](https://github.com/docknetwork/react-native-sdk) enables you to build a Verifiable Credentials wallet inside your app and allows your users to receive, store, and manage their DOCK tokens too. This was built for native applications with added support for Polkadot-JS.
 
 The Dock Wallet SDK handles all the Polkadot Web Assembly in a WebView, sending messages to the React Native thread through a JSON RPC layer.
 
-All you need to do is wrap your app in a `WalletSDKProvider` and start building your Polkadot wallet.
+To use the wallet-sdk, all you need to do is wrap your app in a `WalletSDKProvider` and start building your wallet.
+
+Dock Mobile SDK supports devices that have  Android 8.1 or higher and iOS 11 or higher.
+
+Using [polkadot-js](https://polkadot.js.org/) libraries in React Native is a challenge, due to the lack of WebAssembly support.
```

Comment:
this comment feel incomplete. I was expecting a follow-up sentence explaining how we get around that limitation by using a WebView

Reply:
I just added the comments that we had in our previous wallet SDK documents. Feel free to add missing comments

Reply:
@mike-parkhill I have moved the sentences around to address your comments, but feel free to change. The goal is to have something that is correct for now and get the examples synced as well and if a more extensive description is needed I can add that later.